app/assets/stylesheets/base/_values.scss

```diff
@@ -0,0 +1,16 @@
+// This file contains the base fonts-sizes for the application.
+
+$body: 20px;
```

Comment:
I think `$body` variable name is too general and not so obvious. This file can have more variables in the future. What about `body_font` or something like that?